### ISSUE TITLE
Corretto checksums have changed

### ISSUE THREAD

**Issue by damacus** [sous-chefs/java#607]
### :speaking_head: Foreword

Thank for taking the time to fill this bug report fully. Without it we may not be able to fix the bug, and the issue may be closed without resolution.

## :ghost: Brief Description

Amazon have updated the Corretto packages thus changing the checksums

## :pancakes: Cookbook version

8.0.0

## :woman_cook:  Chef-Infra Version

Version of chef-client in your environment.

## :tophat: Platform details

n/a

## Steps To Reproduce

RUn

```bash
kitchen test corretto-11-amazon
```

## :police_car: Expected behavior

Corretto downloads and installs correctly

## :heavy_plus_sign: Additional context

It I would be nice if we could grab these checksums from the website, but they only provide MD5 hashes right now

**Comment by duncaan** [comment]
Unless chef provides a md5 sum property or AWS provides a sha256 sum, we can't automate this.

**Comment by jakauppila** [comment]
AWS provides an MD5 checksum. So for example, the package

- https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz

hosts an MD5 checksum at 

- https://corretto.aws/downloads/latest_checksum/amazon-corretto-11-x64-linux-jdk.tar.gz

Source: https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html

**Comment by jakauppila** [comment]
Similarly with the checksum values changing, so has the `full_version` value that is defaulted within `corretto_sub_dir`.

So as-is, if you were to utilize the resource as

```ruby
corretto_install '11' do
  action :install
end
```

It'll extract it at `/usr/lib/jvm/java-11-corretto/amazon-corretto-11.0.7.10.1-linux-x64`, but sets `node.default['java']['java_home']` to `/usr/lib/jvm/java-11-corretto/amazon-corretto-11.0.6.10.1-linux-x64`

**Comment by duncaan** [comment]
Opened https://github.com/corretto/corretto-11/issues/125 to formally request checksums that chef can use.

**Comment by damacus** [comment]
Resolved by #656
